loki/transformations/inline.py

```diff
@@ -84,7 +86,8 @@ def __init__(
             inline_internals=False, inline_marked=True,
             remove_dead_code=True, allowed_aliases=None,
             adjust_imports=True, external_only=True,
-            resolve_sequence_association=False
+            resolve_sequence_association=False,
+            creates_edges=False
```

Comment:
Manifest properties shouldn't be a runtime option but instead need to be specified as a class property on the transformation class implementation. But I would vote for setting `creates_items = True` in the manifest instead, as explained above.

Reply:
Yes I agree it would be cleaner to just add it to the manifest. I was just trying to make it optional so we only incur this cost if we have to, I am happy to set `creates_items = True` to the manifest instead.